doc/man1/flux-shell.rst

```diff
@@ -138,6 +138,17 @@ file itself (see INITRC section, ``plugin.register()`` below)
 
 By default, flux-shell supports the following plugin callback topics:
 
+**taskmap.SCHEME**
+  Called when a taskmap scheme *SCHEME* is requested via the taskmap
+  shell option or corresponding ``--taskmap`` ``flux-mini`` option.
+  Plugins that want to offer a different taskmap scheme than the defaults
+  of ``block``, ``cyclic``, and ``manual`` can register a ``taskmap.*``
+  plugin callback and then users can request this mapping with the
+  appropriate ``--taskmap=name`` option. The default block taskmap is
+  passed to the plugin as "taskmap" in the plugin input arguments, and
+  the plugin should return the new taskmap as a string in the output args.
```

Comment:
I didn't catch that the default block taskmap is passed to the plugin when I commented earlier.  The cyclic plugin just calls `flux_shell_get_taskmap()`.

(Verified that the block taskmap is indeed part of the plugin input args - set up in `shell_taskmap()`)

I guess ignore my previous comment!

Reply:
Yeah, `flux_shell_get_taskmap()`  was added after the fact. The first version of the cyclic plugin used the taskmap encoded string in the plugin input args, but I felt after `flux_shell_get_taskmap()` was added, it was silly to decode the string when the builtin plugin could just get the taskmap directly.